### ISSUE TITLE
After latest commit, bufferline is not being hidden when `showtabline=0` while on dashboard.

### ISSUE THREAD

**Issue by Pocco81** [akinsho/nvim-bufferline.lua#146]
Hello @akinsho! The tittle says it all. This started happening since updating the plugin two days ago. I read through the code and [removing `VimEnter`](https://github.com/akinsho/nvim-bufferline.lua/commit/cebafb95622205a414a6c10bf0e40d197cc652b1#diff-47fabfe452ff7a153117c6d9b99f2ef62455cb8ae325dd77fd62d85fe3d10d4dR889) from the event seems to be the problem.

Could you add it back and leave it as it was before?

**Comment by akinsho** [comment]
Thanks for reporting this @Pocco81 I don't use a startup menu and didn't notice a difference in testing so I assumed it was serving no purpose. Does it work as expected for you if the event is re-added?

**Comment by Pocco81** [comment]
Surprisingly enough, adding VimEnter didn't work. However I did revert to the commit 344564a and everything works as expected. Are the changed from the latest commit necessary?

**Comment by akinsho** [comment]
@Pocco81 that's not really necessary should be able to fix it without resorting to such drastic measures

**Comment by Pocco81** [comment]
What can we/you do then?

**Comment by akinsho** [comment]
@Pocco81 I understand what is probably going on and have raised a PR can you try it and report back on there if it fixes the issue.

**Comment by Pocco81** [comment]
Nope, it's still the same :(

This is my config btw:
```
require "bufferline".setup {
    options = {
        mappings = true,
        enforce_regular_tabs = true,
        buffer_close_icon = "",
        modified_icon = "●",
        close_icon = "",
        left_trunc_marker = "",
        right_trunc_marker = "",
        numbers = "ordinal",
        number_style = "superscript",
        diagnostics = "false",
        max_name_length = 14,
        max_prefix_length = 13,
        tab_size = 16,
        view = "multiwindow",
        show_buffer_close_icons = true,
        separator_style = "thin",
        offsets = {{filetype = "NvimTree", text = "Explorer"}},
        always_show_bufferline = false -- hide buffer if there are less than one
    }
    -- highlights = {
    -- 	background = {					-- inactive tab color
    -- 		guifg = comment_fg,
    -- 		guibg = "#282c34"
    -- 	},
    -- 	buffer_selected = {				-- active tab color
    -- 		guifg = normal_fg,
    -- 		guibg = "#3A3E44",
    -- 		gui = "bold"
    -- 	},
    -- 	fill = {						-- bufferline's background color
    -- 		guifg = comment_fg,
    -- 		guibg = "#282c34"
    -- 	},
    -- 	separator_visible = {
    -- 		guifg = "#282c34",
    -- 		guibg = "#282c34"
    -- 	},
    -- 	separator_selected = {
    -- 		guifg = "#282c34",
    -- 		guibg = "#282c34"
    -- 	},
    -- 	separator = {						-- separator color. first one is the thin line; second one is the thick one
    -- 		guifg = "#393b43",
    -- 		guibg = "#282c34"
    -- 	},
    -- 	indicator_selected = {				-- separator when tab is active color
    -- 		guifg = "#1da8f2",
    -- 		guibg = "#3A3E44"
    -- 	},
    -- 	modified_selected = {				-- color when modified right hand side of the tab
    -- 		guifg = string_fg,
    -- 		guibg = "#3A3E44"
    -- 	}
    -- }
}
```

**Comment by Pocco81** [comment]
I'm gonna try to load bufferline after Dashboard to see if that does something.

**Comment by akinsho** [comment]
Can we move the conversation to the PR please, so I can follow on just one thread? Really strange that isn't working. I'll download dashboard see

**Comment by Pocco81** [comment]
Yep.

**Comment by akinsho** [comment]
fixed by #147
